src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -125,14 +125,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 assemblyIdentityComparer:=assemblyIdentityComparer,
                 strongNameProvider:=strongNameProvider,
                 metadataImportOptions:=MetadataImportOptions.Public,
-                referencesSupersedeLowerVersions:=False)
+                referencesSupersedeLowerVersions:=False,
+                ignoreCorLibraryDuplicatedTypes:=False)
 
         End Sub
 
-        Friend Sub New(
+        Private Sub New(
```

Comment:
I don't think so. Is there a reason to prevent internal code from using this constructor? Could they do something wrong? Is there a risk of misuse? It feels like the answer to all these questions is "no".